.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```

Comment:
Title/label questions as above: would default here make sense?

Reply:
Labels vary depending on severity and title is unique to the bug lets not force a default here.

Reply:
I think the label is "enhancement" in this repo and sure we can add that